pkg/cluster_upgrader/cluster_upgrader.go

```diff
@@ -464,7 +464,7 @@ func performUpgradeVerification(c client.Client, logger logr.Logger) (bool, erro
 			}
 		}
 	}
-	if len(dsList.Items) != readyDS {
+	if totalDS != readyDS {
```

Comment:
For anyone wondering about this change - it was a bug I found through writing tests, so I fixed it so that the test could pass. It was considering the count of non-core daemonsets when assessing how many were ready.